lib/datadog/appsec/contrib/graphql/gateway/multiplex.rb

```diff
@@ -55,7 +55,9 @@ def build_arguments_hash
 
             def arguments_from_selections(selections, query_variables, args_hash)
               selections.each do |selection|
-                next unless selection.is_a?(::GraphQL::Language::Nodes::Field)
+                # rubocop:disable Style/ClassEqualityComparison
+                next unless selection.class.to_s == Integration::AST_NODE_CLASS_NAMES[:field]
```

Comment:
If this is intending to get the class name, why not use `selection.class.name`?

Reply:
oh, good catch, sorry, I should use Copilot more carefully